libavfilter/dnn/dnn_backend_native_layer_conv2d.h

```diff
@@ -34,7 +34,36 @@ typedef struct ConvolutionalParams{
     float *biases;
 } ConvolutionalParams;
 
+/**
+ * @brief Load the 2D Convolution Layer.
+ *
+ * It assigns the layer parameters to the hyperparameters
+ * like dilation, padding method, activation, bias, and
+ * kernel size after parsing from the model file context.
+ *
+ * @param layer pointer to the DNN layer instance
+ * @param model_file_context pointer to model file context
+ * @param file_size model file size
+ * @param operands_num number of operands for the layer
+ * @return Size of DNN Layer
+ * @retval 0 if model file context contains invalid hyperparameters.
+ */
 int ff_dnn_load_layer_conv2d(Layer *layer, AVIOContext *model_file_context, int file_size, int operands_num);
+
+/**
+ * @brief Execute the 2D Convolution Layer.
+ *
+ * Possible values for activation are RELU, TANH,
+ * SIGMOID, LEAKY_RELU and NONE.
```

Comment:
I think,  maybe there is no need to specify the possible values of activation function?  Since it's not necessary infomation to call the func.

Reply:
Oh, right. It isn't necessary for execution. I'll remove this line.